src/bin/mrtg

```diff
@@ -241,12 +241,15 @@ sub main {
     # so we lock on the cfg file. Nothing fancy, just a lockfile
 
     $lockfile = $opts{"lock-file"} || $lockfile;
+	my @cfgfile = split(/\//, $cfgfile);
 
     if (! defined $lockfile) {
-        $lockfile = $cfgfile."_l";
+#DEB        $lockfile = $cfgfile."_l";
+       $lockfile = "/var/lock/mrtg/" . join('_', @cfgfile) . "_l";
```

Comment:
this assumes that mrtg is runnning 'on the system' and not as a simple user ... on the system this does make sense, but it should be conditional on the folder begin available and else stay local.